app/src/main/java/io/pslab/fragment/ThermometerDataFragment.java

```diff
@@ -333,6 +333,10 @@ public void saveGraph() {
             try {
                 csvFile.createNewFile();
                 PrintWriter out = new PrintWriter(new BufferedWriter(new FileWriter(csvFile, true)));
+                SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
I wonder why we are rewriting the whole code that's already implemented in `CSVLogger.java` file. `writeCSVFile(String data)` does the same thing. I can see this in many files in this PR